### ISSUE TITLE
Latest version not on packagist.org and repo-structure problematic for composer install

### ISSUE THREAD

**Issue by Crizz0** [s9e/phpbb-ext-highlighter#8]
I tried to update this phpBB extension via `../composer.phar update` within my phpBb installation and run into an issue, that it will not find any newer versions than 1.2.4.

Looked at: https://packagist.org/packages/s9e/phpbb-ext-highlighter and found out, that higher versions are not available, although the releases are here on github.

Additionally, I think the new structure of this repository introduced by v1.5.0 will leads to more problems here as well.

Regards

**Comment by JoshyPHP** [comment]
I published the 2.0.2 tag on GitHub to see if it gets picked up by Packagist.

I don't know anything about installing phpBB extensions with Composer, is there an article about it somewhere?

**Comment by Crizz0** [comment]
I don't think so, but it is really a short story.

You add the extension of your choice here in a new line in the phpBB **composer.json**: https://github.com/phpbb/phpbb/blob/bd1c3c87b7586b41f068a9bb476ae3fd1be1ba6e/phpBB/composer.json#L53 and the `composer/installers`:

This will lead to:
```
	"twig/twig": "^1.0 || ^2.0",
	"composer/installers": "~1.0",
	"crizzo/ipanonym": "~1.0.0"
```

Then you run the following command from the phpBB-root-directory (where the composer.json) is:
`../composer.phar install` (composer.phar is here in the parent directory of phpBB/, like in the original repository: https://github.com/phpbb/phpbb/blob/3.3.x/composer.phar

It will then automatically copy the fitting version (to the version contraint, here 1.0.3) of this extension, by looking it up here: https://packagist.org/packages/crizzo/ipanonym (or any other github, gitlab, gitolite etc. repository if you define them) to the phpBB/ext directory.

Therefor the webhook to: https://packagist.org/api/github needs to be setup for your github repository, how this works was changed 1-2 years ago. Maybe you need to update your config. 

Another problem is afaik the composer.json needs to be in the repository root e.g.: https://github.com/Crizz0/ipanonym

**Comment by JoshyPHP** [comment]
I've just published version 3.0.1 and as far as I can tell, it updates correctly via Composer. Let me know how that works out on your end.

**Comment by Crizz0** [comment]
Looks good now :) 

Thx for helping with this and fixing the issue 👍
